pkg/cluster/cluster.go

```diff
@@ -43,14 +43,28 @@ type Peer struct {
 }
 
 const (
-	DefaultPushPullInterval = 5 * time.Second
-	DefaultGossipInterval   = 5 * time.Second
-	DefaultRefreshInterval  = 60 * time.Second
+	DefaultRefreshInterval = 60 * time.Second
+
+	// Peer's network types. These are used as a predefined peer configurations for a specified network type.
+	LocalNetworkPeerType = "local"
+	LanNetworkPeerType   = "lan"
+	WanNetworkPeerType   = "wan"
+)
+
+var (
+	// AvailableNetworkPeerTypes is a list of available peers' network types.
+	AvailableNetworkPeerTypes = peerNetworkType{LocalNetworkPeerType, LanNetworkPeerType, WanNetworkPeerType}
 )
 
 // PeerType describes a peer's role in the cluster.
 type PeerType string
 
+type peerNetworkType []string
```

Comment:
I don't like it. We created a type just for formatting, but actually you can do `AvailableNetworkPeerTypes = []string{LocalNetworkPeerType, LanNetworkPeerType, WanNetworkPeerType}` and `%v` in `errors.Errorf` just fine.

What do you think?

Reply:
in that case you have to duplicate `strings.Join`, but I don't mind to simplify it.

Reply:
better to duplicate `string.Join` than unnecessary type